theories/_CoqProject

```diff
@@ -1,6 +1,7 @@
 -Q . Undecidability
 
 -arg -w -arg -notation-overridden
+-arg "-set 'Default Proof Using=Type'"
```

Comment:
```suggestion
-arg "-set \"Default Proof Using=Type\""
```
I need to change this line to process files in Proof General. Can you check whether it breaks something on your side again?

Reply:
If I use `-arg "-set \"Default Proof Using=Type\""` as you suggest, then `make` does not compile the project anymore.